Snippets/Core/Core_6/Pipeline/OnReceivePipelineCompleted.cs

```diff
@@ -0,0 +1,25 @@
+namespace Core6.Pipeline
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus;
+
+    class OnReceivePipelineCompleted
+    {
+        public OnReceivePipelineCompleted()
+        {
+            var endpointConfiguration = new EndpointConfiguration("MyEndpoint");
+
+            #region ReceivePipelineCompletedSubscription
+
+            endpointConfiguration.Pipeline.OnReceivePipelineCompleted(e =>
```

Comment:
Initially I wasn't sure, but I like to be explicit. Results in less questions from users. And the page doesn't contain the much information yet, so it's probably better to add it.